lib/matplotlib/widgets.py

```diff
@@ -1100,6 +1113,24 @@ def disconnect(self, cid):
         """Remove the observer with connection id *cid*."""
         self._observers.disconnect(cid)
 
+    @property
+    def rectangles(self):
```

Comment:
This property needs to be deprecated, and you also need to handle `.lines`.

Reply:
Yes. Updated now.